dashboard/app/assets/javascripts/client_state.js

```diff
@@ -32,8 +32,10 @@ var MAX_LINES_TO_SAVE = 1000;
 var COOKIE_OPTIONS = {expires: dashboard.clientState.EXPIRY_DAYS, path: '/'};
 
 dashboard.clientState.reset = function() {
-  $.removeCookie('lines', {path: '/'});
-  sessionStorage.clear();
+  try {
```

Comment:
Add a test in client state tests for this (if possible - not sure how you fake throwing an exception on sessionStorage clear)